.github/workflows/ci.yml

```
@@ -131,18 +131,24 @@ jobs:
       run: docker build . --file ./php/Dockerfile.beta --tag php:$(date +%s)
 
   # SCALA
+  build-scala-latest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: scala-bionic
+      run: docker build . --file ./scala/build/Dockerfile.latest --tag scala-latest:$(date +%s)
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:74: Quote this to prevent word splitting

(shellcheck)